lib/generators/blacklight/sitemaps/templates/solrconfig.xml

```diff
@@ -0,0 +1,294 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<config>
+  <!--for Sematext Monitoring-->
+  <jmx />
+
+  <!-- NOTE: various comments and unused configuration possibilities have been purged
+       from this file.  Please refer to http://wiki.apache.org/solr/SolrConfigXml,
+       as well as the default solrconfig file included with Solr -->
+
+  <abortOnConfigurationError>${solr.abortOnConfigurationError:true}</abortOnConfigurationError>
+
+  <luceneMatchVersion>7.4.0</luceneMatchVersion>
+
+  <directoryFactory name="DirectoryFactory" class="${solr.directoryFactory:solr.NRTCachingDirectoryFactory}"/>
+
+  <!-- solr lib dirs -->
+  <lib dir="${solr.install.dir:../../../..}/contrib/analysis-extras/lib" />
+  <lib dir="${solr.install.dir:../../../..}/contrib/analysis-extras/lucene-libs" />
+
+  <!-- using default data directory here, for the vanilla Solr installation-->
+  <dataDir>${solr.data.dir:}</dataDir>
+
+  <codecFactory class="solr.SchemaCodecFactory"/>
+
+  <schemaFactory class="ClassicIndexSchemaFactory"/>
+
+  <indexConfig>
+    <!-- native is the default, but let's be safe -->
+    <lockType>${solr.lock.type:native}</lockType>
+    
+    <!-- in-memory buffer of 200MB per shard. If it fills up, 
+     a hard commit is done, but the transaction log isn't cleared up,
+     so we still have to rely on the hard commit policy -->
+    <ramBufferSizeMB>200</ramBufferSizeMB>
+  </indexConfig>  
+
+  <updateHandler class="solr.DirectUpdateHandler2">
+
+    <!-- this is our transaction log for SolrCloud-->
+    <updateLog>
+      <str name="dir">${solr.ulog.dir:}</str>
+      <int name="numVersionBuckets">${solr.ulog.numVersionBuckets:65536}</int>
+    </updateLog>
+
+    <!-- hard commit every 10 minutes or 100K docs, whichever comes first -->
+    <autoCommit>
+      <maxTime>600000</maxTime>
+      <maxDocs>100000</maxDocs>
+      <openSearcher>false</openSearcher>
+    </autoCommit>
+
+    <!-- soft commit every 20 minutes -->
+    <autoSoftCommit>
+      <maxTime>1200000</maxTime>
+    </autoSoftCommit>
+
+  </updateHandler>
+
+  <updateProcessor class="solr.processor.SignatureUpdateProcessorFactory" name="add_hash_id">
+    <bool name="enabled">true</bool>
+    <str name="signatureField">hashed_id_si</str>
+    <bool name="overwriteDupes">false</bool>
+    <str name="fields">id</str>
+    <str name="signatureClass">solr.processor.Lookup3Signature</str>
+  </updateProcessor>
+
+  <updateRequestProcessorChain name="cloud" processor="add_hash_id" default="true">
+    <processor class="solr.LogUpdateProcessorFactory"/>
+    <processor class="solr.DistributedUpdateProcessorFactory"/>
+    <processor class="solr.RunUpdateProcessorFactory"/>
+  </updateRequestProcessorChain>
```

Comment:
This is the pertinent part of the solrconfig for this feature. Couple of thoughts about things we could do:

1. Make a kind of solr partial and have rails add it if the chunk is not present (though `updateRequestProcessorChain` would have to have a line inserted in most cases)
2. Take the vanilla solrconfig from Blacklight and just add this part to it 
3. Take this chunk and add it to the README

I know we talked about this on Friday, but I thought I'd put a comment here too. If I'm leaning one way it would be to add it to the README as a manual integration step because sorlconfigs can vary so much.